snf-cyclades-app/synnefo/userdata/migrations/0002_os_keypairs.py

```diff
@@ -0,0 +1,38 @@
+# -*- coding: utf-8 -*-
+from __future__ import unicode_literals
+
+from django.db import models, migrations
+import datetime
+import django.core.validators
+
+
+class Migration(migrations.Migration):
+
+    dependencies = [
+        ('userdata', '0001_initial')
+    ]
+
+    operations = [
+        migrations.CreateModel(
```

Comment:
That's a really good point. That happened because I accidentally squashed all migrations into 1, so the snf-manage script acted accordingly